src/app/showStartingPage.js

```diff
@@ -0,0 +1,112 @@
+                c1.9,14.3,1.9,29.2-0.2,44.9c-0.5,4.3-1.4,8.8-2.1,13.1c-0.7,3.8-1.9,7.6-3.1,11.2c-0.7,1.9-1.4,3.8-2.1,5.7"/>
+                <path fill="#3762AC" stroke="#3762AC" stroke-width="10" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" d="
+                M513.4,507.6c-16.6-2.1-37.6-3.8-59.7-13.1c-23.8-10-17.1-36.1,10.7-35.2c14.7,0.7,30.7,2.6,49.4,3.3
+                C515.6,476.9,515.3,491.9,513.4,507.6z"/>
+                <path fill="#3762AC" stroke="#3762AC" stroke-width="10" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" d="
+                M508.2,531.8c-5.2,15.7-14.5,29.2-29,39.5l0,0c0,0-36.8-16.2-46.4-25.7c-20.9-20.7-6.7-39.5,23.3-32.1
+                C469.4,516.8,508.2,531.8,508.2,531.8z"/>
+            </g>
+            <g id="tail">
+                <path fill="#FFCE06" stroke="#3762AC" stroke-width="10" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" d="
+                M726.9,501.4l-66.6,2.4l-29.7,51.1c-25.2-1-49.2-2.1-72.5-3.1c-1.2,2.4-2.4,4.5-3.3,6.7c-10.2,20-17.8,30.2-23.3,33
+                c-28.1-2.1-50.4-6.2-65.9-12.6c5-2.4,9.5-4.8,13.6-7.6l0,0c12.8-9,21.4-20.4,26.9-34l7.1,1.9c1.7-5,3.6-9.7,5.5-14.5
+                c6.7-17.4,14-33.3,23.8-45.4c8.1-5.2,25.9-4.5,44.7-3.1c7.6-26.9,18.5-73.2,23.8-78.5c11.4-16.4,103.9-41.6,147.4-35.7
+                C764.2,393.2,754.2,439.3,726.9,501.4z"/>
+                <path fill="#3762AC" stroke="#3762AC" stroke-width="10" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" d="
+                M554.8,558.7c-10.2,20-17.8,30.2-23.3,33c-28.1-2.1-50.4-6.2-65.9-12.6c5-2.4,9.5-4.8,13.6-7.6l0,0c12.8-9,21.4-20.4,26.9-34
+                l7.1,1.9c1.7-5,3.6-9.7,5.5-14.5h4.5v4.8h7.8l1,7.4h10.9v12.8h8.6L554.8,558.7z"/>
+            </g>
+            <g id="heart">
+                <path id="heart4" fill="#D61E40" d="M518.9,78.9c-12.1-21.4-32.3-25.9-38.5,0c-3.3,14.3,5.7,27.8,16.2,39.2
+                c6.7,7.1,15.2,8.8,25,4.8c19.3-7.8,34.9-19,34.5-34.5C555.3,65.4,537.2,68.5,518.9,78.9z"/>
+                <path id="heart3" display="none" fill="#D61E40" d="M508.9,123.1c-7.6-13.4-20.2-16.2-24.1,0c-2.1,8.9,3.6,17.4,10.1,24.5
+                c4.2,4.5,9.5,5.5,15.6,3c12.1-4.9,21.9-11.9,21.6-21.6C531.6,114.6,520.3,116.5,508.9,123.1z"/>
+                <path id="heart2" display="none" fill="#D61E40" d="M500.9,145.3c-5.7-10-15.2-12.2-18.1,0c-1.6,6.7,2.7,13.1,7.6,18.4
+                c3.1,3.3,7.1,4.1,11.7,2.2c9-3.7,16.4-8.9,16.2-16.2C518,138.9,509.5,140.4,500.9,145.3z"/>
+                <path id="heart1" display="none" fill="#D61E40" d="M490.4,163.7c-3.8-6.7-10.1-8.1-12.1,0c-1,4.5,1.8,8.7,5.1,12.3
+                c2.1,2.2,4.8,2.8,7.8,1.5c6-2.5,10.9-6,10.8-10.8C501.8,159.5,496.2,160.4,490.4,163.7z"/>
+            </g>
+    `;
+
+    appScreen.innerHTML += pikachuSvgImage.innerHTML;
+
+    // add leaderboard modal
```

Comment:
Można to zastąpić po prostu metodą `addLeaderboardModal` i komentarz jest zbędny.